assets/sass/utils/_helpers.scss

```diff
@@ -38,6 +38,10 @@ $LAYOUT
     float: right;
 }
 
+.rtl-button-fix {
```

Comment:
Sorry I am late on this! We don't need this extra class, we can make use of the same methodology we do in other places in the platform-client. You can read about it here: http://preview.ushahidi.com/platform-pattern-library/master/assets/html/front-end-guidelines/read-direction.html. 

So what we need to do is figure out the right and left positioning of the button in this piece of code: https://github.com/ushahidi/platform-pattern-library/blob/master/assets/sass/5_layouts/_layout-d.scss#L42.

You can see how we handle this in the map-view here: https://github.com/ushahidi/platform-pattern-library/blob/master/assets/sass/4_blocks/_toolbar.scss#L27

I hope this gives you an idea of how to solve it. However well done figuring it out the way you did, this code-base is large and quite complex so its not easy :) Let me know if you need more guidance!